### PR TITLE
Add registerWith* properties for register template

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

### What does this PR do?
Fix user registration template.

### Screenshot/screencast of this PR
Before:
Screenshot from 2020-09-17 09-57-59

After:
Screenshot from 2020-09-17 10-35-17

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14083

### How to test this PR?
1. Build keycloak image
2. Use keycloak image in keycloak deployment

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by tolusha** [comment]
[crw-ci-test]

**Comment by tolusha** [comment]
[crw-ci-test]

File: dockerfiles/keycloak/che/login/messages/messages_en.properties [hunk 1]
```diff
@@ -6,3 +6,6 @@ formRegister=Create an account with a form
 passwordRecover=Forgot your password? <br/> Enter your email below.
 
 usernameTitle=Username has to comply with the DNS naming convention. An alphanumeric (a-z, and 0-9) string, with a maximum length of 63 characters, with the '-' character allowed anywhere except the first or last character.
+
```
Comment (skabashnyuk) [inline dockerfiles/keycloak/che/login/messages/messages_en.properties]: do we need new lines between properties?

**Review by skabashnyuk — APPROVED** [review]

File: dockerfiles/keycloak/che/login/messages/messages_en.properties (hunk 1 above)
Comment (tolusha) [reply dockerfiles/keycloak/che/login/messages/messages_en.properties]: no, removed.

**Comment by tolusha** [comment]
[crw-ci-test]

**Comment by tolusha** [comment]
The only failed test is `Language_server_validation_Java_LS_initialization`, related issue https://github.com/eclipse/che/issues/17864

**Pull request merged**